src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/IntegrationIds.cs

```diff
@@ -0,0 +1,29 @@
+// ReSharper disable InconsistentNaming - Name is used for integration names
+namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
+{
+    internal enum IntegrationIds
+    {
+        HttpMessageHandler,
```

Comment:
I hadn't noticed this before but this looks like that to configure each instrumentation one needs to add it here. Ideally, this should code from the integrations.json. Anyway, not related to this PR per se.

Reply:
There is a env var that enables / disables integrations without modifying integrations.json. The name here and in Integrations.json must match.